src/Microsoft.AspNetCore.Mvc.RazorPages/ApplicationModels/PageApplicationModel.cs

```diff
@@ -16,26 +19,25 @@ public class PageApplicationModel
         /// <summary>
         /// Initializes a new instance of <see cref="PageApplicationModel"/>.
         /// </summary>
-        /// <param name="relativePath">The application relative path of the page.</param>
-        /// <param name="viewEnginePath">The path relative to the base path for page discovery.</param>
-        public PageApplicationModel(string relativePath, string viewEnginePath)
+        public PageApplicationModel(
+            PageActionDescriptor actionDescriptor,
+            TypeInfo pageType,
+            TypeInfo modelType,
+            TypeInfo handlerType,
+            IReadOnlyList<object> handlerAttributes)
```

Comment:
yeah or maybe just the route template as a string. It's mostly for informational purposes.